src/components/JobsPanel/jobsPanel.util.js

```diff
@@ -297,3 +326,13 @@ export const generateRequestData = (
     }
   }
 }
+
+export const parseDefaultDataInputsContent = inputs => {
+  return inputs.reduce((prev, curr) => {
+    return {
+      ...prev,
+      [curr.data.name]:
+        curr.data.path.pathType + [curr.data.path.url || curr.data.path.value]
```

Comment:
Do you mean:
```suggestion
        curr.data.path.pathType + (curr.data.path.url || curr.data.path.value)
```
?

I don't see a reason to concatenate a string to an array that has a single string in it:
```js
'pathType' + ['url']
// or
'pathType' + ['value']
```